css/main.css

```diff
@@ -850,3 +850,15 @@ media queries
     margin: 0 10px;
   }
 }
+/* =================================
+Team Code Page Styles
+==================================== */
+.teamcode_aside-navigation li{
```

Comment:
@24sint it looks like these changes were made in another PR as well: https://github.com/team-refcode/refcode.org/pull/157/files#diff-23b024677d4732ee53b6d7bdc52e0fe2R853-R864

Once you merge the other PR can you update this with the most recent changes from the `master` branch? Let me know if you have any questions or want to walk through the process in class.

Reply:
@cocarson Thank you for your feedback and approval. That is right, I have more PR it's not approved, and merged yet, and I also updated the **index.html** file.  I merged the team code file PR, and the **master** branch is now up to date,  but I have two more new branches which need approval and to merge on the master.